package.json

```diff
@@ -74,9 +75,8 @@
   "peerDependencies": {
     "@lingui/core": "5.x",
     "@warp-ds/css": "2.1.1",
-    "@warp-ds/elements-core": "2.x",
     "lit": "3.x",
-    "react": "^19.1.1"
+    "react": "^18.x"
```

Comment:
Maybe react should not be a peerDependency here as the team who is only using this repo with Lit, would get react in their node modules because of this :)

Reply:
yeah makes sense!

Reply:
I'll remove all peer deps. We just want the end users to just install the packages and get intellisense so no need to make them install Lit or Warp CSS etc either.